modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
This would involve changes to the rest of the codebase where these values are fetched and created.  If this is a worthwhile change I am willing to make it across the whole codebase, but I don't think it belongs in this PR.  This values are defined in external_connect.js and then referenced in a few other places in the code.